js/src/util.js

```diff
@@ -332,7 +332,10 @@ const elementCanHaveAltText = (tagName) => {
   }
 
   const tagNameLowerCase = tagName.toLowerCase()
-  return tagNameLowerCase === 'img' || tagNameLowerCase === 'image' || tagNameLowerCase === 'area'
+  return tagNameLowerCase === 'img' ||
+    tagNameLowerCase === 'image' ||
+    tagNameLowerCase === 'area' ||
+    tagNameLowerCase === 'canvas'
```

Comment:
It seems like some sort of array.include method might be cleaner here

Reply:
done.